Vizzuality/gfw
# feat/otf-analysis

## Overview

Introducing new OTF analysis service for un-cached widget data.

## Notes

This service utilizes [this](https://data-api.globalforestwatch.org/#operation/zonal_statistics_get_analysis_zonal__geostore_id__get) API in its core and documentation of the actual implementation can be found [here](https://www.notion.so/OTF-analysis-service-59d1539be54e43b799ae3a783c56bcf7). 

To be done: 

1. Implement remaining data structures inside `data/otf-data.js`
2. Implement new service in all widgets that need OTF data. currently `forest-loss` implemented.

## Testing

- Ensure data is correct for new implementation (validate API) 
- Ensure widgets render correctly when OTF is needed.
